string tid
float64 range
float64 bearing_angle
